### PR TITLE
Issue #6586: aligned javadoc/xdoc for HiddenField

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6586 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/xdocs/config_coding.xml [hunk 1]
```diff
@@ -1365,24 +1365,26 @@ public class MyClass {
...
-              <pre>
-                class Foo {
-                    int prop;
-                    Foo setProp(int prop) {
-                        this.prop = prop;
-                        return this;
-                    }
-                }
-              </pre>
+              <source>
```
Comment (rnveach) [inline src/xdocs/config_coding.xml]: Won't changing this to `source` inside the property table put a block box around it? Wouldn't that look a bit confusing?
I also question having such examples in property table and not being in the description in somewhere else.

File: src/xdocs/config_coding.xml [hunk 2]
```diff
@@ -1365,24 +1365,26 @@ public class MyClass {
...
             <td>
+              Allow to ignore the parameter of a property setter
+              method returning class in which setter is defined.
               Used in conjunction with <code>ignoreSetter</code> property it
               controls rule that recognizes method as a setter. By default
               setter is a method with signature of type
-              <pre>
-                void setXyz(${someType} xyz)
-              </pre>
+              <source>
```
Comment (rnveach) [inline src/xdocs/config_coding.xml]: same.

File: src/xdocs/config_coding.xml [hunk 3]
```diff
@@ -1365,24 +1365,26 @@ public class MyClass {
           <tr>
             <td>setterCanReturnItsClass</td>
             <td>
+              Allow to ignore the parameter of a property setter
+              method returning class in which setter is defined.
               Used in conjunction with <code>ignoreSetter</code> property it
               controls rule that recognizes method as a setter. By default
```
Comment (rnveach) [inline src/xdocs/config_coding.xml]: Would a better description be: `Allow to expand the definition of a setter method to include methods that return the class' instance`
?

**Comment by pbludov** [comment]
@rnveach I'm afraid this documentation can't just be aligned. This needs to be completely redone.

**Comment by pbludov** [comment]
@rnveach, please start the review again.

**Comment by rnveach** [comment]
Thanks @pbludov , I think it looks alot better.

**Pull request merged**